### ISSUE TITLE
Mixed languages

### ISSUE THREAD

**Issue by MatBe14** [raithos/xwing#800]
### Expected Behavior

I select English

### Actual Behavior

I get a mix of French and English. It's mostly in English, but some pilots are shown in French when I load a squad, and the grey text for available (empty) slots is displayed in French.

### Language

English

**Comment by raithos** [comment]
what language is your browser set to? French?

**Comment by MatBe14** [comment]
I use Chrome and everything is displayed in English.

**Comment by MatBe14** [comment]
OK, I noticed that when I click on my bookmark to load YASB in Chrome, I first see "Français" for maybe 2 seconds, then it switches to English. Maybe that's why? Is there a setting somewhere that changed automatically maybe, either in Chrome or YASB? Because I didn't change anything for sure.

**Comment by MatBe14** [comment]
OK, yeah, I checked following your instructions and it says that my Chrome has 3 languages (!): "en-us", "en" and "fr". I accessed the Advanced Settings in Chrome and found the Languages, removed French and now the issue seems to be gone. Sorry about any wasted time over this, I just didn't realize what languages were set in the browser. Maybe it's automatic? I don't remember setting any myself anyway. I'm bilingual and write in French daily, so maybe it's automatically recognized?
